.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+<!--- TITLE -->
+
+#### Module/Algorithm
+- [ ] Core algorithm/module implementation
+- [ ] [Query module](https://memgraph.com/docs/memgraph/reference-guide/query-modules) implementation
+- [ ] Unit tests
+- [ ] End-to-end tests
+- [ ] Code documentation
+- [ ] README short description
+- [ ] Documentation on [memgraph/docs](https://github.com/memgraph/docs)
+
+#### Bug/Enhacement
+
+<!--- Write the problem description and propose how to solve it -->
+
+- [ ] Code documentation
```

Comment:
I also don't know if this is necessary. But on the other hand, we can put to update README.